molecule/provisioner/lint/ansible_lint.py

```diff
@@ -29,55 +29,7 @@
 LOG = logger.get_logger(__name__)
 
 
-class AnsibleLint(base.Base):
-    """
-    `Ansible Lint`_ is the default role linter.
-
-    `Ansible Lint`_ checks playbooks for practices, and behaviour that could
-    potentially be improved.
-
-    Additional options can be passed to `ansible-lint` through the options
-    dict.  Any option set in this section will override the defaults.
-
-    .. code-block:: yaml
-
-        provisioner:
-          name: ansible
-          lint:
-            name: ansible-lint
-            options:
-              exclude:
-                - path/exclude1
-                - path/exclude2
-              x: ["ANSIBLE0011,ANSIBLE0012"]
-              force-color: True
-
-    The `x` option has to be passed like this due to a `bug`_ in Ansible Lint.
-
-    The role linting can be disabled by setting `enabled` to False.
-
-    .. code-block:: yaml
-
-        provisioner:
-          name: ansible
-          lint:
-            name: ansible-lint
-            enabled: False
-
-    Environment variables can be passed to lint.
-
-    .. code-block:: yaml
-
-        provisioner:
-          name: ansible
-          lint:
-            name: ansible-lint
-            env:
-              FOO: bar
-
-    .. _`Ansible Lint`: https://github.com/ansible/ansible-lint
-    .. _`bug`: https://github.com/ansible/ansible-lint/issues/279
-    """
+class AnsibleLintMixin(object):
```

Comment:
Could you please add additional copyright line to the top of this file?

Reply:
Also, we can avoid having to explicitly inherit from object:
```suggestion
__metaclass__ = type


class AnsibleLintMixin:
```